docs/user_guide/mitigation.rst

```diff
@@ -111,7 +111,9 @@ The amount of correlation that is removed can be controlled using the
 
 Note that the lack of correlation does not imply anything about statistical dependence.
 Therefore, we expect this to be most appropriate as a preprocessing step for
-(generalized) linear models.
+(generalized) linear models. Furthermore, since correlation measures linear
```

Comment:
I would move the "Furthermore, since correlation [...] in the data." *before* "Therefore, we expect [...] linear models." and replace "Furthermore" by "In particular"